### DISCUSSION TITLE
Expo plugins for Storage/Database?

### DISCUSSION THREAD

**Discussion by Nantris** [invertase/react-native-firebase#7368] [Ideas] [answered]
Are these planned or managed via community contributions?

Thanks for all the great work Invertase team!

**Comment by mikehardy** [comment, marked as answer]
I don't believe those packages - database and storage - have any need for extra native configuration and thus I do not believe that there is any need for expo config plugins.

They should already work fine with Expo

Do you have some reason to believe differently? 🤔

**Reply by Nantris** [reply]
I was under the impression they'd be required. I got an error when including the (non-existent) plugins in our `app.config.js` but I see now that it probably would have worked fine if I'd just removed the plugins from the list.

If I find that not to be the case I'll update here. Thanks @younes0 and @mikehardy for your replies.

**Reply by BoavistaLudwig** [reply]
@Nantris did you npx expo install @react-native-firebase/storage and got it working? I thought we need a plugin if a package comes with native code? I have to rebuild my custom dev app and am currently getting a failed build because of ... SwiftCompile normal arm64 /Users/expo/workingdir/build/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageDownloadTask.swift (in target 'FirebaseStorage' from project 'Pods') ...

**Reply by Nantris** [reply]
@BoavistaLudwig yes it worked for us without any plugin. I'm not sure about your error, sorry.
